charts/connectors/templates/tests/01-mm2-values.yaml

```diff
@@ -143,6 +143,8 @@ spec:
 
           curl {{ template "curl-options" . }} -H 'Content-Type: application/json' http://{{ include "connectors.serviceName" . }}:{{ .Values.connectors.restPort }}/connectors -d @/tmp/mm2-conf.json
 
+          sleep 30
```

Comment:
Let's add a comment here about why we're sleeping before merging. If there's a known error message, it would be good to include it in the comment as well.

Reply:
Good idea I will add short explanation.